src/Pages/HomePage/HomePage.css

```diff
@@ -0,0 +1,56 @@
+:root{
+    --bg-color: #fcfcfc;
+    --category-color:#e7e2e2;
```

Comment:
use color variables instead of direct values

Reply:
but this is the root.